ufo2fontir/src/source.rs

```diff
@@ -696,9 +708,14 @@ impl Work<Context, WorkId, WorkError> for StaticMetadataWork {
             .instances
             .iter()
             .map(|inst| NamedInstance {
-                name: match inst.name.strip_prefix(family_prefix.as_str()) {
+                name: match inst
+                    .name
```

Comment:
I just noticed that we are using the instance `name` attribute to set the fvar's NamedInstance's subfamilyNameID names, but this is wrong, varLib uses the `stylename` attribute of an instance, not its `name` for that purpose. The latter is similar to source.name and only meant to uniquely identify a given DS source/instance element for cross referencing within the doc (should have been called `id` perhaps but it's too late).

https://github.com/fonttools/fonttools/blob/57fb47de3a41c724c3bf1051d00715e08d86141e/Lib/fontTools/varLib/__init__.py#L103-L118

If we use `stylename` here we no longer need to do inst.name.strip_prefix(family_prefix).